aptos-move/framework/aptos-stdlib/sources/hash.move

```diff
@@ -14,4 +14,17 @@ module aptos_std::aptos_hash {
         // TODO: temporary mockup.
         pragma opaque;
     }
+
+    struct HashValue has copy, drop, store{
```

Comment:
use it for table handle as well?

Reply:
that's the plan, I need to wait https://github.com/move-language/move/pull/391 landed and upgrade the dependency